.github/workflows/build_gradle_sam.yml

```diff
@@ -0,0 +1,30 @@
+name: di-gradle-sam
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'gradle-sam/*'
+      - '.github/workflows/build_gradle_sam.yml'
+  registry_package:
+    types:
+      - 'updated'
```

Comment:
I don't know but I think we want "published" - I thought that "published" == "new image push", "updated" = "someone tweaked the release notes or metadata for an existing image". Really unsure on this though 🤷

Reply:
Yeah I have no idea, we'll go with published if its wrong we can amend it